src/main/java/ch/njol/skript/effects/EffClearEntityStorage.java

```diff
@@ -0,0 +1,51 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.EntityBlockStorage;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Clear Entity Storage")
+@Description("Clear the stored entities of an entity block storage (i.e. beehive).")
+@Examples("clear the stored entities of {_beehive}")
+@RequiredPlugins("Paper")
+@Since("INSERT VERSION")
+public class EffClearEntityStorage extends Effect {
+
+	static {
+		if (Skript.methodExists(EntityBlockStorage.class, "clearEntities"))
+			Skript.registerEffect(EffClearEntityStorage.class,
+				"(clear|delete) the entity storage of %blocks%");
```

Comment:
```suggestion
				"(clear|empty) the entity storage of %blocks%");
```
I don't think delete's a good word here, since it implies the storage itself is being deleted, rather than emptied/cleared.